inject.js

```diff
@@ -626,31 +645,9 @@ function setSpeed(controller, video, speed) {
   var speedvalue = speed.toFixed(2);
   video.playbackRate = Number(speedvalue);
   refreshCoolDown();
-  rateChanged(controller);
```

Comment:
This call is no longer needed now that the ratechange blocker (to stop "bouncing" playback rate) and ratechange listener (for updating the controller's speed value) have been combined into one. setSpeed inherently calls the logic in rateChanged by virtue of the listener.